ams/server/src/main/java/com/netease/arctic/server/persistence/converter/TaskRewriteInputMapHandler.java

```diff
@@ -0,0 +1,66 @@
+package com.netease.arctic.server.persistence.converter;
+
+import com.google.common.collect.Maps;
+import com.netease.arctic.optimizing.RewriteFilesInput;
+import com.netease.arctic.utils.SerializationUtil;
+import org.apache.ibatis.type.BaseTypeHandler;
+import org.apache.ibatis.type.JdbcType;
+
+import java.sql.CallableStatement;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Types;
+import java.util.Map;
+
+public class TaskRewriteInputMapHandler extends BaseTypeHandler<Map<Integer, RewriteFilesInput>> {
```

Comment:
Would extending `Object2ByteArrayConvert` make the implementation of this class simpler?

Reply:
Yes, i did.